test/Microsoft.ML.AutoML.Tests/UserInputValidationTests.cs

```diff
@@ -33,10 +33,13 @@ public void ValidateExperimentExecuteNullLabel()
         [Fact]
         public void ValidateExperimentExecuteLabelNotInTrain()
         {
-            var ex = Assert.Throws<ArgumentException>(() => UserInputValidationUtil.ValidateExperimentExecuteArgs(Data,
-                new ColumnInformation() { LabelColumnName = "L" }, null, TaskKind.Regression));
+            foreach (var task in new[] { TaskKind.Recommendation, TaskKind.Regression })
```

Comment:
I would have passed down TaskKind as arg using Theory/InlineData but because it is an internal type, I went against that idea.